site/content/usage/11-customizing-the-kubelet.md

```diff
@@ -45,7 +45,14 @@ nodeGroups:
         evictionHard:
             memory.available:  "200Mi"
             nodefs.available: "10%"
+        featureGates:
+            DynamicKubeletConfig: true
 ```
 
 In this example, given instances of type `m5a.xlarge` which have 4 vCPUs and 16GiB of memory, the `Allocatable` amount
- of CPUs would be 3.4 and 15.4 GiB of memory. 
+of CPUs would be 3.4 and 15.4 GiB of memory. In addition, the `DynamicKubeletConfig` feature gate is also enabled. It is
+important to know that the values specified in the config file for the `featureGates` will completely overwrite the
+default values specified by eksctl, like `RotateKubeletServerCertificate`, so if you still want to keep it you need to
```

Comment:
I'd make this a little more concrete, e.g.:

```markdown
IMPORTANT: by default `eksctl` sets  **`featureGates.RotateKubeletServerCertificate`** to `true`, when custom **`featureGates`** are provide, it will be unset. You should always include **`featureGates.RotateKubeletServerCertificate`**, unless you have to disable it.
```